prost-derive/src/field/scalar.rs

```diff
@@ -696,7 +696,7 @@ impl DefaultValue {
             DefaultValue::String(ref value) => quote!(#value.to_owned()),
             DefaultValue::Bytes(ref value) if value.is_empty() => quote!(::std::vec::Vec::new()),
             DefaultValue::Bytes(ref value) => {
-                let lit = LitByteStr::new(value, Span::def_site());
+                let lit = LitByteStr::new(value, Span::call_site());
```

Comment:
call_site() will mean what def_site() meant previously, everything is unhygienic (visible from outside the macro). The def_site() will be hygienic, invisible from outside. But that is not currently possible on stable, that's a nightly-only feature. Macros 2.0 will be finalized before Rust 2018 (so in the next few months) and after that it will be possible to write hygienic macros too.